### DISCUSSION TITLE
The use of imitation learning examples, can not be trained to the ideal effect

### DISCUSSION THREAD

**Discussion by qiangl9618** [isaac-sim/IsaacLab#684] [Q&A]
### Question

 I collected 200 demo data in the mock example, grabbed the object and put it in the specified position, And then use " ./isaaclab.sh -p source/standalone/workflows/robomimic/train.py --task Isaac-Lift-Cube-Franka-IK-Rel-v0 --algo bc --dataset logs/robomimic/Isaac-Lift-Cube-Franka-IK-Rel-v0/hdf_dataset.hdf5 " to do the training, after the completion of the training, always unable to achieve the expected effect.
I would like to know if you have experimented with examples of imitation learning, and could you share your experience

**Comment by alexinrobotics** [comment]
I also have the same question. It seems that the basic bc is not good for the task. I think they should add more information about how to use the robomimic(imitation learning) part in the isaac lab! BTW, I also try to use the bc_transformer for this task, struggling to set the self.algo.transformer.context_length  in bc_config.py. Do you have any idea of that?
